src/views/tutorial-view/components/next-previous/components/directional-link-box/index.tsx

```diff
@@ -1,46 +1,38 @@
 import React from 'react'
-import Link, { LinkProps } from 'next/link'
-import InlineSvg from '@hashicorp/react-inline-svg'
+import classNames from 'classnames'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
+import { IconSliders24 } from '@hashicorp/flight-icons/svg-react/sliders-24'
+import CardLink from 'components/card-link'
+import Text from 'components/text'
+import { DirectionalLinkBoxProps, DirectionOption } from './types'
 import s from './directional-link-box.module.css'
 
-interface DirectionalLinkBoxProps {
-  link: LinkProps
-  label: string
-  title: string
-  direction: DirectionOption
+const IconDict: { [k in DirectionOption]: typeof IconArrowRight16 } = {
+  next: IconArrowRight16,
+  previous: IconArrowLeft16,
+  final: IconSliders24,
 }
 
-type DirectionOption = 'next' | 'previous' | 'final'
-
-const IconSrcDict: { [k in DirectionOption]: string } = {
-  next: require(`@hashicorp/flight-icons/svg/arrow-right-16.svg?include`),
-  previous: require(`@hashicorp/flight-icons/svg/arrow-left-16.svg?include`),
-  final: require(`@hashicorp/flight-icons/svg/sliders-24.svg?include`),
-}
-
-/*
- * @TODO style to spec
- * look at the `CardLink` component to use for base styles!
- */
-
 function DirectionalLinkBox({
-  link,
+  href,
   label,
-  title,
+  ariaLabel,
   direction,
 }: DirectionalLinkBoxProps) {
+  const Icon = IconDict[direction]
+
   return (
-    <Link href={link.href} as={link.as}>
-      <a className={s.linkbox} data-direction={direction}>
-        <span className={s.icon} data-direction={direction}>
-          <InlineSvg src={IconSrcDict[direction]} />
-        </span>
-        <span className={s.text}>
-          <span aria-hidden="true">{label}</span>
-          <span className={s.hidden}>{title}</span>
-        </span>
-      </a>
-    </Link>
+    <CardLink
+      className={classNames(s.linkbox, s[`direction-${direction}`])}
+      href={href}
+      ariaLabel={ariaLabel}
+    >
+      <Icon className={classNames(s.icon, s[`direction-${direction}`])} />
+      <Text asElement="span" size={200} weight="medium">
```

Comment:
Maybe check on the text color and weight, it seems a bit different than the design?

Reply:
✅ Great call - the color was a off, needed to set it to `var(--token-color-foreground-primary)`.

Re: font-weight, I think it's correct should be `500` in both Figma and the browser. I think possibly Figma's rendering is a little different? I do notice that if we disable `-webkit-font-smoothing: antialiased;` on body, then we get a look that seems more closely matched to Figma.